src/utils/helpers.ts

```diff
@@ -100,12 +70,33 @@ export function formatDate(date: Date | string, includeTime: boolean): string {
   return DateTime.fromJSDate(jsDate, { zone: 'utc' }).toFormat(format);
 }
 
-export function getSpecficCvssScore(scores: any[], issuer: string, version: string) {
-  return scores.find(
-    score => score.issuer === issuer && score.cvss_version === version,
-  );
+export function isCVSS3issuedByRH(score: ZodAffectCVSSType) {
+  return score.issuer === 'RH' && score.cvss_version === CVSS_V3;
+}
+
+export function affectRhCvss3(affect: ZodAffectType) {
+  return affect.cvss_scores.find(isCVSS3issuedByRH);
+}
+
+type WithModuleComponent = {
+  ps_component: string;
+  ps_module: string;
+  uuid?: null | string;
+};
+
+export function matchModuleComponent(first: WithModuleComponent, second: WithModuleComponent) {
+  return first.ps_component === second.ps_component && first.ps_module === second.ps_module;
+}
+
+export function doAffectsMatch(first: ZodAffectType, second: ZodAffectType) {
+  const doIdsMatch = first.uuid && second.uuid && (first.uuid === second.uuid);
+  return doIdsMatch || matchModuleComponent(first, second);
+}
+
+export function isAffectIn(affect: ZodAffectType, affects: ZodAffectType[]) {
+  return Boolean(affects.find(affectToMatch => doAffectsMatch(affect, affectToMatch)));
 }
 
-export function getRhCvss3(scores: any[]) {
-  return getSpecficCvssScore(scores, 'RH', 'V3');
+export function affectsMatcherFor(affect: ZodAffectType) {
```

Comment:
What about naming it something like `matchingAffects`?, (sounds a bit clearer to me)

Reply:
Since it returns a function, I find that a name that includes `matcher` is more semantic, but I agree it could be less awkwardly named and have opted for `matcherForAffect`--is this better?